configs/grafana/dashboards/Autometrics Function Explorer.json

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
the reason for that is a grafana dashboard provisioning bug that they refuse to fix for multiple years. as a solution for that, they [recommend](https://github.com/grafana/grafana/issues/10786#issuecomment-392377975) using raw json instead of export for sharing one, which is what i did here